rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -330,6 +326,38 @@ private void addTracingKeys(NodeMsgTraceInfo nodeMsgTraceInfo) {
         }
     }
 
+    private void logStart(MessageTask task) {
+        MessageType messageType = task.getMessage().getMessageType();
+
+        if (task.getNodeMsgTraceInfo() == null) {
+            logger.trace("Start {} message task", messageType);
+            return;
+        }
+
+        long taskWaitTimeInSeconds = task.getNodeMsgTraceInfo().getLifeTimeInSeconds();
+        logger.trace("Start {} message task after {}s", messageType, taskWaitTimeInSeconds);
+
+        if (taskWaitTimeInSeconds >= QUEUED_TIME_TO_WARN_LIMIT) {
+            logger.debug("Task {} was waiting too much in the queue: [{}] seconds", messageType, taskWaitTimeInSeconds);
+            recentDelays = true;
+        }
+    }
+
+    private void logEnd(MessageTask task, long startMillis) {
+        long processTimeInSeconds = (System.currentTimeMillis() - startMillis) / 1000;
+
+        Message message = task.getMessage();
+        boolean isBlockRelated = message.getMessageType() == MessageType.BLOCK_MESSAGE || message.getMessageType() == MessageType.BODY_RESPONSE_MESSAGE;
+        boolean isBlockSlow = isBlockRelated && BlockUtils.tooMuchProcessTime(processTimeInSeconds);
+        boolean isOtherSlow = !isBlockRelated && processTimeInSeconds > PROCESSING_TIME_TO_WARN_LIMIT;
+
+        if (isBlockSlow || isOtherSlow) {
+            loggerMessageProcess.warn("Message[{}] processing took too much: [{}] seconds.", message.getMessageType(), processTimeInSeconds);
+        }
+
+        logger.trace("End {} message task after {}s", task.getMessage().getMessageType(), processTimeInSeconds);
```

Comment:
do you think it'd make sense to wrap values in square brackets in the Start/End longs? for consistency, as it's done in other logs here